### ISSUE TITLE
How to ignore the "#" which the <h1> -<h6>generate

### ISSUE THREAD

**Issue by feiglex** [Alir3z4/html2text#185]
- Version by `html2text --2016.9.19`
- Python version `python --2.7.13`
- `
import html2text

html = "&lt;p&gt;hello, this is &lt;em&gt;html2text&lt;/em&gt;&lt;/p&gt;&lt;strong&gt;it is strong label&lt;/strong&gt;"
h = html2text.HTML2Text()
print h.handle(html)

h.ignore_emphasis = True
print h.handle(html)

html = "&lt;h6&gt;This is title&lt;/h6&gt;"
print h.handle(html) #    value: ###### This is title`

**Comment by lroolle** [comment]
This is a ``html2markdown`` :XD cause we're encountering with the same problem...

**Comment by Alir3z4** [comment]
So you want to keep the whole thing same but only removing `#` from the output ?

**Comment by feiglex** [comment]
yeah, I just want to keep the text i see in the website. with removing #. 
Are there any configuration to remove # ?

**Comment by Alir3z4** [comment]
No, there's no option to disable the formats. We aim to generate a text that at least is able to get back to its original format, removing all format options won't be helpful.

However, for the purpose of removing all the HTML tags from the text, you can use lxml or beautifulsoap and perform stripping html tags that will give.

There's a similar feature request: https://github.com/Alir3z4/html2text/issues/170

**Comment by mj-dd** [comment]
this should be called html2markdown

**Comment by bjd-pfq** [comment]
@mj-dd 

> this should be called html2markdown

Well said. This is a con: it cannot perform what its name implies.
